onnx/common/ir.h

```diff
@@ -767,11 +767,19 @@ struct Node : public Attributes<Node> {
   }
 };
 
+struct OpSetID {
+  std::string domain;
+  int64_t version;
+};
+
 struct Graph final {
 ONNX_DISALLOW_COPY_AND_ASSIGN(Graph);
 friend struct Node;
 friend struct Value;
 
+public:
+  OpSetID opset_version;
```

Comment:
ModelProto can have a list of opset_versions: https://github.com/onnx/onnx/blob/33e9cd4182fe468675241fba4ae8a16c2f0bd82f/onnx/onnx.proto#L198